### DISCUSSION TITLE
Using JS for gamedev...

### DISCUSSION THREAD

**Discussion by gapiyka** [HowProgrammingWorks/LiveQA#20] [Q&A]
Станет ли когда-нибудь JS юзабельным для создания AAA (triple-A) игр? 
———
И вернётся ли популярность разрабатывать игры под веб?
———
![gif](https://media.giphy.com/media/hqTguNdEoA1ooYxeog/giphy.gif)

**Comment by tshemsedinov** [comment]
Да вообще js норм для игр, вот есть квейк на js: http://www.quakejs.com/
